wrapanapi/systems/openstack.py

```diff
@@ -1255,3 +1255,22 @@ def download_object(self, container_name, object_name, path):
 
         with open(path, "wb") as obj:
             obj.write(obj_contents)
+
+    def get_quota(self, quota='all', project=None):
+        """Get quota details.
+        Args:
+            quota: quota type ("all" or "compute" or "volume" or "network"). Default set to "all".
+            project: name of project. Default set to the tenant value of the constructor.
+        Return: quota_values (dict of quota).
+        """
+        project = project or self.tenant
+        quota = str(quota).lower()
+        project_id = self._get_tenant(name=project)
+        quota_values = {}
+        if quota in ['all', 'compute']:
```

Comment:
I think these should be nested under a quota name key, because their keys could overwrite each other with the updates when 'all' is passed.  I don't see any conflicting keys now other than `id`, which is identical in value, but I would like to avoid the possibility here.

Note that there is some structure already to `'network'`, it has a `'quota'` key at its top level, whereas volume/compute does not.

example:
```
In [7]: p.get_quota(quota='compute')                                                                                                        
Out[7]: 
{'injected_file_content_bytes': 10240,
 'metadata_items': 128,
 'server_group_members': 10,
<trunc>}

In [8]: p.get_quota(quota='network')                                                                                                        
Out[8]: 
{'quota': 
    {'subnet': 100,
     'network': 100,
     'floatingip': 50,
<trunc>}}
```

How about returning a dictionary with structure, like:
```
{
    'compute': {<compute quota key/value},
    'network': {<network quota key/value},
    'volume': {<volume key/value>}
}
```

I also think its a bit misleading to use 'project' instead of 'tenant' since we're defaulting it to the System tenant attribute.  Could this just be a property on the System class, instead of a method with kwargs, and the caller can then just look at which key they want, instead of passing it. This means it would only look at the System tenant, and you would have to update the tenant attribute if you want to change what quota is fetched.

With a cursory search I only see one other function on the system that take tenant as a kwarg to filter the results. I think this as a property, only using `self.tenant`, would be a good pattern.

Reply:
Done. I have choose "project" as "Tenant" is the old term and is deprecated from keystone API v3.
>> https://wiki.openstack.org/wiki/Tenant